src/Uno.UI/WasmCSS/Uno.UI.css

```diff
@@ -231,3 +231,7 @@ textarea {
   resize: none;
   overflow: hidden; /* Scrolling is handled by the parent ScrollViewer */
 }
+
+.uno-mirror, .uno-mirror *:not(:first-child:last-child) {
+  transform: scaleX(-1);
+}
```

Comment:
Found this CSS trick which applies the class to all children (only when there are at least 2 children). This looks much easier than having to listen to children changing in C#.